action.yml

```diff
@@ -20,6 +20,8 @@ outputs:
     description: 'version of terraform'
   terragrunt:
     description: 'version of terragrunt'
+  hugo:
+    description: "json object of hugo version e.g. { version: 'v0.100.0', is_extended: false }"
```

Comment:
extended_0.100.0 is converted to `{ version: '0.100.0', is_extended: true }`, so v here can be misleading.